### PR TITLE
Fix go get issue with "unknown revision v0.0.0"

### LINKED ISSUE #1197: go get fails with "unknown revision v0.0.0"
Running go get with go modules enabled fails due to this error:

```
$ GOFLAGS="" GO111MODULE=on go get github.com/GoogleCloudPlatform/spark-on-k8s-operator                                    
go: github.com/GoogleCloudPlatform/spark-on-k8s-operator@v0.0.0-20210307184338-1947244ce5f4 requires
        k8s.io/apiextensions-apiserver@v0.0.0: reading k8s.io/apiextensions-apiserver/go.mod at revision v0.0.0: unknown revision v0.0.0
````

trying just the API package also gives the same error:

```
$ GOFLAGS="" GO111MODULE=on go get github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1beta2 
go: finding github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io/v1beta2 latest
go: finding github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg latest
go: finding github.com/GoogleCloudPlatform/spark-on-k8s-operator latest
go: finding github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis/sparkoperator.k8s.io latest
go: finding github.com/GoogleCloudPlatform/spark-on-k8s-operator/pkg/apis latest
go get: github.com/GoogleCloudPlatform/spark-on-k8s-operator@v0.0.0-20210307184338-1947244ce5f4 requires
        k8s.io/apiextensions-apiserver@v0.0.0: reading k8s.io/apiextensions-apiserver/go.mod at revision v0.0.0: unknown revision v0.0.0
```

### PR DESCRIPTION
**Pull request by hongshaoyang**
Closes #1197 .

### CODE REVIEW

**Comment by hongshaoyang** [comment]
@googlebot I signed it!

**Comment by akhurana001** [comment]
@liyinan926  PTAL . ran into this as well.

**Comment by Ameralameri** [comment]
How work

File: go.mod [hunk 1]
```diff
@@ -25,7 +25,7 @@ require (
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
 	k8s.io/api v0.19.6
-	k8s.io/apiextensions-apiserver v0.0.0
```
Comment (YT026) [inline go.mod:28]:
```
~~
~~
```

**Review by YT026 — APPROVED** [review]

**Pull request merged**